tree_sitter_language_pack/__init__.py

```diff
@@ -194,8 +197,23 @@ def get_binding(language_name: SupportedLanguage) -> object:
     try:
         module = import_module(name=f".bindings.{language_name}", package=__package__)
         return cast(object, module.language())
-    except ModuleNotFoundError as e:
-        raise LookupError(f"Language not found: {language_name}") from e
+    except (ModuleNotFoundError, ImportError) as e:
+        package_path = Path(__file__).parent
+
+        ext = ".pyd" if sys.platform.startswith("win") else ".so"
```

Comment:
why would we need to manually do this? I don't quite understand. `import_module` should take care of this.

Reply:
thats weird. Ok, please add a code comment explaining this buggy behavior. Might be a bug in python itself to file.